Last-Update: 2014-03-07
From: Jan Christoph Nordholz <hesso@pool.math.tu-berlin.de>
Description:
 Set /etc/auto.master as the default location of the automounter
 master map, because that file is surely available (the package
 installs a default one). If the admin already has an auto.master
 map in a different place, he can customize the file later.

--- a/samples/autofs.conf.default.in
+++ b/samples/autofs.conf.default.in
@@ -4,9 +4,9 @@
 [ autofs ]
 #
 # master_map_name - default map name for the master map.
 #
-#master_map_name = auto.master
+master_map_name = /etc/auto.master
 #
 # timeout - set the default mount timeout in secons. The internal
 #	    program default is 10 minutes, but the default installed
 #	    configuration overrides this and sets the timeout to 5
